.travis.yml

```diff
@@ -21,6 +21,14 @@ before_install:
   - sudo apt-get install clang --quiet --yes
   ## GCC.
   - sudo apt-get install build-essential --quiet --yes
+  ## Buck.
+  - sudo apt-get install ant python git
+  - git clone https://github.com/facebook/buck.git
```

Comment:
Yeah, there is no built version of buck for Linux/Windows so you have to build it yourself…